lib/handle-pull-request-change.js

```diff
@@ -5,7 +5,7 @@ async function handlePullRequestChange (robot, context) {
   const isWip = containsWIP(title) || await commitsContainWIP(context)
   const status = isWip ? 'pending' : 'success'
 
-  console.log('Updating PR "%s" (%s): %s', title, html_url, status)
+  console.log(`Updating PR "${title}" (${html_url}): ${status}`)
```

Comment:
unfortunately that breaks the linting with standard:
https://travis-ci.org/gr2m/wip-bot/builds/341085085#L1400

can you change line 4 to 

```js
const {title, html_url: htmlUrl, head} = context.payload.pull_request
```

and the line above to 

```js
console.log(`Updating PR "${title}" (${htmlUrl}): ${status}`)
```

Reply:
Ah, my bad, didn't check the build.
Fixed it. :rocket: